src/widget/form/chatform.cpp

```diff
@@ -1085,5 +1085,47 @@ void ChatForm::hideNetcam()
 
 void ChatForm::retranslateUi()
 {
+    QString volObjectName = volButton->objectName();
+    QString micObjectName = micButton->objectName();
     loadHistoryAction->setText(tr("Load chat history..."));
+
+    switch(callButton->objectName())
+    {
+    case QStringLiteral("green"):
```

Comment:
That comes natural to switch-case statement. It doesn't work with `"literals"` (or other dynamic types) -> must be enumeration type (int, enum, …).